package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "radash",
-  "version": "10.7.0",
+  "version": "10.7.1",
```

Comment:
For a documentation fix we don't want to change the library version. We'll need to revert this.

Reply:
reverted this 👍